jcl/src/java.base/share/classes/openj9/internal/tools/attach/target/DiagnosticUtils.java

```diff
@@ -495,13 +495,19 @@ public void run() {
 				}
 			} else if (command.equalsIgnoreCase(DIAGNOSTICS_JFR_STOP)) {
 				if (VM.isJFRRecordingStarted()) {
+					if (setFileName) {
```

Comment:
I don think we need these. The `filename` param should be ignored completely for non JFRstart commands

Reply:
Ignored the changing of the recording filename within the commands `JFR.dump` and `JFR.stop`.